### ISSUE TITLE
Why you have log of output multiply to y here like reduce_sum(yj*logpj)!

### ISSUE THREAD

**Issue by nooralahzadeh** [unreliableXu/TensorFlow_RLRE#28]
https://github.com/unreliableXu/TensorFlow_RLRE/blob/38370c8a77bbf352ae0328b981cb483cf3850cc6/cnnmodel.py#L83

**Comment by unreliableXu** [comment]
Sorry, I think it is a mistake. It should be :
self.cross_loss = -tf.reduce_mean(tf.reduce_sum(  self.input_y * tf.log( self.outputs),axis=1))
